services/api/mw2pdf/src/classes/MediaWikiSession.ts

```diff
@@ -1,22 +1,25 @@
 import puppeteer from 'puppeteer'
 import {
   PdfGenerator
-} from './PdfGenerator.js'
```

Comment:
I'm so happy to see `.js` lines go away.

Reply:
Me, too!!